src/stylesheets/base/_typography.scss

```diff
@@ -29,6 +29,18 @@
     letter-spacing: none;
   }
 }
+
+.a-sup {
+  @include text($primary-font, normal, 1rem, $color-orange-red);
```

Comment:
A couple of questions/notes here:
- do you need all of this CSS here? This mixin is adding 4 lines of CSS
- does this need to be a class? `sup a {}` seems pretty specific
- does the `sup` still have any default styles that will/can help you here?
- is this link style any different from your other in-body-copy link styles?

Reply:
- removed the class.
- deleted mixin.
- yes, the sup link style is different from others.
resolved  in pr#4